src/wasm.h

```diff
@@ -1335,10 +1335,25 @@ class ArrayLen : public SpecificExpression<Expression::ArrayLenId> {
 // Globals
 
 struct Importable {
+  Name name;
```

Comment:
Not optimal that this is on `Importable`, as this is not really a property of an imported thing, semantically.

However, you are using `Importable` essentially as "global thing on the module", and this would make sense there. So maybe the problem is that `Importable` needs to be renamed. But can be separate from this PR of course.

Reply:
Indeed, see my earlier comment.   The wasm-c-api uses "extern" for this to mean something that can be imported and exports to/from the module.   "extern" and "external" are bit overloaded but I think maybe the spec uses those terms too.. which which case we should rename as a followup.